.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -38,6 +38,6 @@ about: Report a bug to help us improve Istio
 
 **Environment where bug was observed (cloud vendor, OS, etc)**
 
-Additionally, please consider attaching a [cluster state archive](http://istio.io/help/bugs/#generating-a-cluster-state-archive) by attaching
-the dump file to this issue.
-
+Additionally, run `istioctl bug-report` and attache the dump file to this issue.
+Refer [cluster state archive](http://istio.io/help/bugs/#generating-a-cluster-state-archive)
+for more details.
```

Comment:
```suggestion
Additionally, please consider running `istioctl bug-report` and attach the generated cluster-state tarball to this issue. This command is only available in `istioctl` version 1.8.0 and higher, but a newer version of `istioctl` can be used against an older Istio version to run the command and collect this information.
```

Reply:
> Additionally, please consider running `istioctl bug-report` and attach the generated cluster-state tarball to this issue.

Keeping the link to docs because the doc is also going to be updated soon and we can mention that `istioctl bug-report` is available from `1.8.0` in there?

Reply:
That works as long as you update the doc. However, I do think since this command is only available in version 1.8.0 or higher, mentioning it here doesn't hurt as users will have bad UX if they run the command with an older `istioctl` version.